css/about.css

```diff
@@ -189,7 +189,7 @@ body {
 .my-main-logo {
   border: 1px solid #d3d3d3;
```

Comment:
- [ ] There is a screenshot that is not showing correctly in your readme file 📸. Please also record a video about your project max 5 mins and add it to also readme file 🎥. I recommend using loom app for this